app/schemas.py

```diff
@@ -216,9 +214,6 @@ def service_permissions(self, service):
     def get_reply_to_email_address(self, service):
         return service.get_default_reply_to_email_address()
 
-    def get_sms_sender(self, service):
-        return service.get_default_sms_sender()
-
     def get_letter_contact(self, service):
```

Comment:
Does the same apply to letter_contact?

Reply:
used in admin here - https://github.com/alphagov/notifications-admin/blob/master/app/main/views/service_settings.py#L663